### ISSUE TITLE
The spotlights have no light.

### ISSUE THREAD

**Issue by CatDogFishFrog** [bolderbrush10/SearchlightAssault#6]
Version 2.3.6.

When a searchlight shines on an area, it just displays “searchlight-haze.png” (maybe), and that's it. Judging by the illustrative videos, there should be light.

I'm playing with the DarkNight mod, so the lack of light on the searchlights is not convenient at all, because they are simply not visible.

I also tried to run the game exclusively with the SearchlightAssault mod, but it didn't work either.

![image](https://github.com/user-attachments/assets/013b42a8-5a81-4d08-ac6d-39e2c7cf0c9d)

**Comment by bolderbrush10** [comment]
Yeah the light's been turned off for now because of base game API updates.
My first priority has been just keeping the mod playable, I'll put in some kind of placeholder light soon, probably this weekend.


The Factorio 2.0 API update to require graphics_set also seems to have made beams face the wrong direction, so the light was really distracting -- it was constantly facing the "wrong way". There were some other things affected, that I had to turn off for now, like the energy_glow animation for safe-mode. I'm still thinking about how to create a minimum reproduction so I can post it to the bug reports, when I get some time.

**Comment by bolderbrush10** [comment]
This was fixed a while ago, forgot to close the issue.
